### DISCUSSION TITLE
Handling Glances with offline machines

### DISCUSSION THREAD

**Discussion by Jimbo222** [hassio-addons/app-glances#511] [Q&A] [answered]
Hi,

I am starting to play around with Glances and have it running monitoring a number of my machines at home.

I have a question around how best to handle the integrations in HA for monitored devices that are not running 24/7. When those machines are powered off I get a nasty looking failed status on the integration status in HA and a bunch of failed messages in the logs as obviously it cannot communicate with the machine.

Wondering is there is a best practice to handle this more gracefully?

A thought I had was on shutdown of glances on the monitored machine could trigger the integration to 'pause' when a normal shutdown is detected, and an 'unpause' on the next startup.  But obviously that would require a change in the app, and there are some scenarios that would need to be thought out.

**Comment by frenck** [comment, marked as answer]
Your question is about the integration, not the add-on. Sounds like you are mixing up the integration vs add-on terminology.

You probably should ask this question on the Home Assistant Discord chat or their community forums.


../Frenck
